### ISSUE TITLE
Cannot save data from gis editor for spatial column

### ISSUE THREAD

**Issue by MoonE** [phpmyadmin/phpmyadmin#17018]
### Describe the bug
Cannot save data from gis editor for spatial column.

### To Reproduce

1. Go to the insert form on a table with 'Point' column (or any other spatial column)
2. Click on 'Edit/Insert' for the Point column
3. Click 'Copy' to get `'POINT(0 0),' 0` into the form
4. Click 'Save'
5. It tries to execute an invalid query: ``INSERT INTO `test` (`id`, `point`) VALUES (NULL, '\'POINT(0 0)\',0')``
 Instead of ``INSERT INTO `test` (`id`, `point`) VALUES (NULL, ST_PointFromText('POINT(0 0)',0))``

### Expected behavior

A new row should be inserted without error

### Screenshots
![image](https://user-images.githubusercontent.com/4426597/126069880-97001fa4-35ae-4242-b20c-9744574f1efd.png)

### Server configuration

- Operating system:
- Web server: nginx/1.14.2 
- Database version: 8.0.18 - MySQL Community Server - GPL 
- PHP version: 7.4.21 
- phpMyAdmin version: RELEASE_5_1_1-1058-gf0bb3c42e0 

### Client configuration

- Browser: Firefox 90
- Operating system: Linux

### Additional context

Spatial functions are missing from the list of selectable functions when inserting data, may be related.

**Comment by williamdes** [comment]
Hi @MoonE 

Please have a look to the home page because "RELEASE_5_1_1-1058-gf0bb3c42e0" does not indicate the true release as said on https://github.com/phpmyadmin/phpmyadmin/issues/14813#issuecomment-449661474

**Comment by MoonE** [comment]
Isn't the last part of that string the commit hash of the version? `f0bb3c42e0`
What info do you need?

**Comment by williamdes** [comment]
> Isn't the last part of that string the commit hash of the version? `f0bb3c42e0`
> What info do you need?

Do not worry it is okay for me, but for contributors it may be more complicated to understand this is why I advise to say the real version

**Comment by williamdes** [comment]
On MariaDB 10.6 I am able to use the insert by selecting the function after the fix of #17079

```sql
INSERT INTO `gis_all` (`name`, `shape`) VALUES ('', ST_GeomFromText('POINT(0 0)',0));
```

**Comment by williamdes** [comment]
Maybe the fix would be to select a geometry function after copying the value
